client/app/styles/onboarding.styl

```diff
@@ -5,6 +5,12 @@
 @import "./base/_colors.styl"
 
 
+// Icons
+
+[data-type=password] svg
+    background-color: #4dcec5
```

Comment:
If we need to add custom bacjground-colors behing SVG icons, I prefer to not hadrcode them in CSS, which may stay generics, and prefer to directly inline background-color in a style attribute in the view template (using a model property to store the color)